index.css

```diff
@@ -29,53 +30,70 @@ a {
 	padding: 0 1.8vw 0 1.5vw;
 }
 
-/******** grid ***********/
+/********* social media logos ******/
+
+
+.socialWrapper {
+	display:flex;
+	justify-content:flex-end;
+	float:none;
+}
+
+.socialWrapper > a {
+	/* social links */
+	float: right;
```

Comment:
i wouldn't mix flex and floats. actually maybe the float can be straight out deleted here